src/lib/y2partitioner/widgets/encrypt.rb

```diff
@@ -107,6 +115,22 @@ def display_encrypt_method?
         controller.methods.size > 1
       end
 
+      # Help texts for available encryption methods
+      #
+      # @return [String]
+      def help_for_methods
+        texts = controller.methods.map do |m|
```

Comment:
You can simply:

```
text = send("help_for_#{m.id}", m)
"<li>#{text}</li>"
```

Reply:
Right! I don't understand why I added the not necessary line of code there :/